python/pylibcudf/pylibcudf/tests/test_nvtext_stemmer.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pylibcudf as plc
+import pytest
+from utils import assert_column_eq
+
+
+@pytest.fixture(scope="module")
+def input_col():
+    arr = ["trouble", "toy", "sygyzy"]
+    return pa.array(arr)
+
+
+@pytest.mark.parametrize("check_vowels", [True, False])
+@pytest.mark.parametrize("indices", [[3, 1, 4], 1])
+def test_is_letter(input_col, check_vowels, indices):
+    def is_letter(s, i, check):
+        vowels = ["a", "e", "i", "o", "u", "y"]
+        if check:
+            if s[i] in vowels:
```

Comment:
Remove the "return True" / etc. as we already have the result.
```suggestion
            return s[i] in vowels
```